### ISSUE TITLE
Fluid results 0.006 m3

### ISSUE THREAD

**Issue by Xenom555** [TheFarawayDev/Better_Recipes_x10#7]
![Showcase](https://github.com/user-attachments/assets/173f9459-8e9c-424f-855e-5fc73fc720de)

Hi, Dev

Thank you for fixing issues with Fluid. 
I noticed that results on a lot recipes is 0.006 m3. Can you look on it i give you screenshot on 4 recipes. Maybe this happen on all Fluid recipes. Fuel, Turbofuel, Residual fluid is essential prod. for factory. Maybe you can keep it as "vanilla" to dont be soo much overpowered because we are limited on pipe size. so if you make it too overpower then 1 refinery can block whole pipeline.

Thank you

**Comment by TheFarawayDev** [comment]
I plan to add either a phase system or a configuration to set the recipes output and inputs! And I will work on this now!

**Comment by TheFarawayDev** [comment]
If not I will balance such resources in the future or up the pipes handle, really depends on user feedback which is why I appreciate this kind of feedback!

Your problem has been resolved!

Thanks for your help, Dev.

**Comment by Xenom555** [comment]
@TheFarawayDev Good idea. But fluids that are produced that will be consumed in another recipe are hard to balance. For example if you produce too much residual oil but your next recipe require only 6m3 on input then you create issue whit all refinery setup. so i think all this kind of fluids need to have some balance.

**Comment by TheFarawayDev** [comment]
I will try to balance, likely outcome is that Crude Oil will be the one upclocked to require more as it would be the best way to shorten the others as well if that doesn't solve issues I will make all fluids *5 of the other input if any.

Hopefully I will succeed and I appreciate the feedback that you have provided and I really mean it when I say your opinions do sway the mods outcome in the future as I care more about your and many others hopes for the mods as it will lead the mod to have a better user feedback.

Anyways thanks for your help that is the summary of this!

**Comment by Xenom555** [comment]
@TheFarawayDev 
I think this recipe still have 0.06 m3 
![image](https://github.com/user-attachments/assets/2744511a-91a6-4936-a7a0-6f501e77a468)
![image](https://github.com/user-attachments/assets/45184136-7e3d-4279-88b1-b8b696ee2c0f)
![image](https://github.com/user-attachments/assets/90412383-3642-421d-86b6-ca9a4bbb0d48)
Thank you

**Comment by Xenom555** [comment]
> I will try to balance, likely outcome is that Crude Oil will be the one upclocked to require more as it would be the best way to shorten the others as well if that doesn't solve issues I will make all fluids *5 of the other input if any.
> 
> Hopefully I will succeed and I appreciate the feedback that you have provided and I really mean it when I say your opinions do sway the mods outcome in the future as I care more about your and many others hopes for the mods as it will lead the mod to have a better user feedback.
> 
> Anyways thanks for your help that is the summary of this!

So regarding all fuel you can look how many m3 consumes Fuel-Generator power this is end of consumtion fuel, turbofuel, biofuel so base on that you can balance whole Fuel. so for example 1 refinery can support 10 Fuel-Generator without overclocking.

**Comment by TheFarawayDev** [comment]
I understand, I will down lock outputs to half that!

**Comment by Xenom555** [comment]
![image](https://github.com/user-attachments/assets/34a200af-ca23-4500-98ce-ee0d75ac9472)
![image](https://github.com/user-attachments/assets/b3f313a9-6a1f-4dd6-90a6-58e93b010299)
![image](https://github.com/user-attachments/assets/81b2abaa-4e6f-481d-bfa1-38251737d183)

**Comment by TheFarawayDev** [comment]
I followed [this](https://docs.google.com/spreadsheets/d/1YZeW4P480ZqZmvvhM6JR6tTQ_1iq_hFpFYO2whcrWGA/edit?gid=0#gid=0) primarily. That was by the original mod maker, I followed it piece by piece but I added only the new stuff and modified some recipes, though I plan to change some recipes most users feel it should stick to the sheet and not be changed. Though I am still collecting data the chances of change may only happen to some recipes (Fuel, Batteries) other than that many would dislike of the mod as a whole!

And I do understand where you are coming from and I agree in some form which is why I met a compromise of sorts, but it seems many users like having pipes backed up so they don't have to worry about running out of something.
